RecoLocalTracker/Phase2TrackerRecHits/python/Phase2StripCPEESProducer_cfi.py

```diff
@@ -2,5 +2,8 @@
 
 phase2StripCPEESProducer = cms.ESProducer("Phase2StripCPEESProducer",
   ComponentType = cms.string('Phase2StripCPE'),
-  parameters    = cms.PSet()
+  parameters    = cms.PSet(
+    LorentzAngle_DB = cms.bool(False),			
+    TanLorentzAnglePerTesla = cms.double(0.07)
```

Comment:
Yes, both P in PS, S in PS, and S in 2S are covered.
True, phase2OTCPE would be much better. Several of the file names in that package are not well chosen... If the powers that be prefer, this can of course be changed, but maybe now is not the best moment.